js/whitebit.js

```diff
@@ -615,11 +706,437 @@ module.exports = class whitebit extends Exchange {
+            fee = {
+                'cost': this.parseNumber (dealFee),
+                'currency': feeCurrencyCode,
+            };
+        }
+        const timestamp = this.safeTimestamp2 (order, 'ctime', 'timestamp');
+        const lastTimestamp = this.safeTimestamp (order, 'ftime');
+        return this.safeOrder2 ({
+            'info': order,
+            'id': orderId,
+            'symbol': symbol,
+            'clientOrderId': clientOrderId,
+            'timestamp': timestamp,
+            'lastTradeTimestamp': lastTimestamp,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'type': type,
+            'stopPrice': activationPrice,
+            'amount': amount,
+            'filled': filled,
+            'fee': fee,
+            'cost': undefined,
+            'trades': undefined,
+        }, market);
+    }
+
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'orderId': parseInt (id),
+        };
+        let market = undefined;
+        if (symbol !== undefined) {
+            market = this.market (symbol);
+            request['market'] = market['id'];
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit; // default 50, max 100
+        }
+        const response = await this.v4PrivatePostTradeAccountOrder (this.extend (request, params));
+        // {
+        //     "records": [
+        //         {
+        //             "time": 1593342324.613711,      // Timestamp of executed order
+        //             "fee": "0.00000419198",         // fee that you pay
+        //             "price": "0.00000701",          // price
+        //             "amount": "598",                // amount in stock
+        //             "id": 149156519,                // trade id
+        //             "dealOrderId": 3134995325,      // completed order Id
+        //             "clientOrderId": "customId11",  // custom order id; "clientOrderId": "" - if not specified.
+        //             "role": 2,                      // Role - 1 - maker, 2 - taker
+        //             "deal": "0.00419198"            // amount in money
+        //         }
+        //     ],
+        //     "offset": 0,
+        //     "limit": 100
+        // }
+        const data = this.safeValue (response, 'records', []);
+        return this.parseTrades (data, market);
+    }
+
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'ticker': currency['id'],
+        };
+        let method = 'v4PrivatePostMainAccountAddress';
+        if (this.isFiat (code)) {
+            method = 'v4PrivatePostMainAccountFiatDepositUrl';
+            const provider = this.safeNumber2 (params, 'provider');
```

Comment:
I think here it has to be the opposite, safeNumber instead of safeNumer2, cause there's just one key. The same of typo is repeating a few times below, should be pretty easy to fix, though.